webview/js/api.py

```diff
@@ -66,6 +64,12 @@
                     // Full file path support for WebView2
                     if (params.event instanceof Event && params.event.type === 'drop' && params.event.dataTransfer.files) {
                         chrome.webview.postMessageWithAdditionalObjects('FilesDropped', params.event.dataTransfer.files);
+                        let event = {};
+                        for (let i in params.event) {
+                            if (i === 'currentTarget') continue;
+                            event[i] = params.event[i];
+                        }
+                        params.event = event
```

Comment:
What is purpose of this code?

Reply:
if this page has many dom, `pywebview._stringify(params)` will take a few seconds. so we remove `currentTarget`

Reply:
is currentTarget the reason for poor performance?
You could probably simplify this by `delete params.event.currentTarget`

Reply:
yes, `currentTarget` is  the reason for poor performance.
I have tried `delete params.event.currentTarget` and `params.event.currentTarget=null`, they do not work.